scripts/syncup-s3.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+# This script is used to sync files required for training module notebooks to S3
```

Comment:
Do you expect this will only get run once? I think this question is better phrased: If we add new data, how do you anticipate folks doing the development will get it onto S3? If it will be by adding to this script, we should probably plan to add something to the contributing guidelines.

Reply:
I expect it to get run any time an input file is added or changed. Which is to say, I definitely think something about this (and discussion of `make-live.R` & GHA) needs to be added to the contributing guidelines. 

I have a followup PR that adds the GHA part (partly). I will file that draft shortly for discussion, as it has some limitations/flaws that we will need to discuss.